site/samples/jdbc.md

```diff
@@ -0,0 +1,42 @@
+## JDBC Sample
+
+This sample WDT domain model section includes configuration for two JDBC data sources.
+ 
+The data source `datasource-1` has a sparse configuration meant to set a minimal number of values. It is not necessary to include additional attributes and sub-folders, unless you intend to override the default values.
+
+The data source `datasource-2` includes additional folders and attributes to provide further customization, including targeting information, connection pool configuration, and such. The cluster `cluster-1` should be defined elsewhere within this model, or should already exist in a domain that is being updated.  
+
+It is recommended that credential fields, such as `PasswordEncrypted`, should not be stored as clear text in the model. Those values can be referenced in a separate variables file or Kubernetes secret, or the model can be encrypted using the [Encrypt Model Tool](../encrypt.md).
+
+```yaml
+resources:
+    JDBCSystemResource:
+        'datasource-1':
+            JdbcResource:
+                DatasourceType: GENERIC
+                JDBCDriverParams:
+                    URL: 'jdbc:oracle:thin:@//localhost:1521/myDB'
+                    DriverName: oracle.jdbc.OracleDriver
+        'datasource-2':
```

Comment:
Why 2 datasources one without any Connection pool properties and one with?  Would it not be better to have only one with the Connection Pool properties?  If you are going to have 2 then have maybe a GENERIC and A GridLink DS.